docs/frost-textarea.md

```diff
@@ -15,6 +15,7 @@ a text area component
 | | | `true` | disabled text area |
 | `form` | `string` | `<form-owner>` | form element that this is associated with (its form owner) |
 | `hook` | `string` | `<unique-name>` | name used for testing with ember-hook |
+| `onClear`| `function`/`action`| empty function | triggers associated action when `X` is clicked and text is cleared |
```

Comment:
Similar documentation for `frost-button` is

onClick | string | <action-name> | triggers associated action when the button is clicked